tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -4,6 +4,8 @@
 class BugzillaLite(Base):
 
     name = 'Bugzilla Lite'
+    manifest_url = "app://bzlite.gaiamobile.org/manifest.webapp"
```

Comment:
This would need the  self.DEFAULT_PROTOCOL and self.DEFAULT_APP_HOSTNAME treatment, I would think.